### ISSUE TITLE
Folder tree view & use workspace packages name

### ISSUE THREAD

**Issue by timotheeguerin** [vitest-dev/vscode#267]
**Is your feature request related to a problem? Please describe.**
When working in a monorepo with many packages and many tests having all the files flattened makes it a little hard to navigate as well as running all the test for a specific package or area.

**Describe the solution you'd like**
It would be nice if
1. It would include the file structure in the tree view. So instead of having 
  ```
  packages/foo/src/area/subarea1/foo.test.ts/
     test1
     test2
  
  packages/foo/src/area/subarea2/foo.test.ts/
     test1
     test2
  ...
  ```
  we could have 
  ```
  packages/
    foo/
     src/
       area/
         subarea1/foo.test.ts/
           test1
           test2
         subarea2/foo.test.ts/
           test1
           test2
  ...
  ```
2. In the case of a vitest workspace it could be nice to have the top level be the vitest project name to reduce nesting( as this name must be unique in the workspace and is used when running vitest on the workspace it feels right to me). 
  From previous example:
  ```
    foo/
     src/
       area/
         subarea1/foo.test.ts/
           test1
           test2
         subarea2/foo.test.ts/
           test1
           test2
  ...
  ```

**Describe alternatives you've considered**

**Additional context**
Those 2 pieces are connected but could be treated as 2 different feature request if you prefer.

**Comment by vovsemenv** [comment]
Really want this feature. 
With current tree its nearly impossible to navigate if you have a lot of tests
![Uploading image.png…]()

**Comment by sheremet-va** [comment]
This is fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.
